Task_5.3.py

```diff
@@ -0,0 +1,16 @@
+# Создайте функцию генератор чисел Фибоначчи
+
+def fibonacci(n: int):
+    first, second = 1, 1
+    for i in range(n):
+        if i == 0:
+            yield first
+        elif i == 1:
+            yield second
+        else:
+            first, second = second, first + second
+            yield second
```

Comment:
Нужно было сделать генератор всех чисел Фибоначчи, а не только `n` первых. В любом случае там кода на 5 строк. Не нужно усложнять. Вот вам код на генерацию `n` первых чисел ряда Фибоначчи. Исправьте его на для генерации всех чисел ряда
```suggestion
def fibonacci(n: int):
    first, second = 0, 1
    for _ in range(n):
        yield first
        first, second = second, first + second
```